vhost-device-vsock/src/vsock_conn.rs

```diff
@@ -174,11 +174,21 @@ impl<S: AsRawFd + Read + Write> VsockConnection<S> {
                         pkt.set_op(VSOCK_OP_RW).set_len(read_cnt as u32);
 
                         // Re-register the stream file descriptor for read and write events
-                        VhostUserVsockThread::epoll_register(
+                        if VhostUserVsockThread::epoll_modify(
                             self.epoll_fd,
                             self.stream.as_raw_fd(),
                             epoll::Events::EPOLLIN | epoll::Events::EPOLLOUT,
-                        )?;
+                        )
+                        .is_err()
+                        {
+                            if let Err(e) = VhostUserVsockThread::epoll_register(
+                                self.epoll_fd,
+                                self.stream.as_raw_fd(),
+                                epoll::Events::EPOLLIN | epoll::Events::EPOLLOUT,
+                            ) {
+                                info!("epoll_register failed: {:?}, but proceed further.", e);
```

Comment:
Maybe `error!`?
At the end is something that should not happen.

Reply:
Since we are here, in send_pkt() we do something similar, but using `.unwrap()`, what about putting the log also there (in another commit)?

Reply:
made it error! and handled send_pkt as well in the following commig.